sdk/communication/AzureCommunicationChat/Source/ChatClient.swift

```diff
@@ -192,4 +194,62 @@ public class ChatClient {
             }
         }
     }
+
+    /// Start receiving realtime notifications.
+    /// Call this function before subscribing to any event.
+    /// - Parameter completionHandler: Called when starting notifications has completed.
+    public func startRealTimeNotifications(completionHandler: @escaping (Result<Void, AzureError>) -> Void) {
```

Comment:
@JoshuaLai @tjprescott just calling out this change - to address the async `credential.token` call in init() I moved initializing of signaling client into here & added the completion handler, `self.signalingClient` is optional again